themes/classic/_dev/css/theme.scss

```diff
@@ -56,8 +57,8 @@
   }
 }
 
-/*** HEADER ***/
+/*** HELPERS ***/
 
-.relative{
+._relative{
```

Comment:
If we follow the [rscss.io](http://rscss.io/variants.html) guidelines, I think it should be `.-relative`